Analyses/src/StoppedParticlesDumper_module.cc

```diff
@@ -50,21 +50,26 @@ namespace mu2e {
       float z;
       float t;
       float tau; // proper time, for stopped pion weights
+      int pdg;
+      int origin;
+      int term;
 
-      StopInfo() : x(), y(), z(), t(), tau() {}
+      StopInfo() : x(), y(), z(), t(), tau(), pdg(0), origin(0), term(0) {}
```

Comment:
I do not understand the purpose of the changes to StoppedParticlesDumper.  The sole purpose of the module was to provide inputs for stopped particle resampling; it is not a "general use" ntuple dumper.  With migration to the framework based resampling the logical thing would be to retire and delete this module, not to extend it.

Reply:
This module is an analyzer, I extended its use as an analyzer, in order to study the output, which is what analyzers are supposed to do.  Its use in production is discontinued.  Perhaps the confusion is that there are still scripts that reference it in JobConfig, i will delete those.

Reply:
Thanks